### ISSUE TITLE
about discriminator loss

### ISSUE THREAD

**Issue by JeromeYHJ** [leftthomas/SRGAN#69]
why discriminator loss set:
d_loss = 1 - real_out + fake_out
is it same as the paper?
why not use torch.nn.BCELoss() to calculate Cross Entropy?
anyway thx

**Comment by leftthomas** [comment]
@JeromeYHJ 
Duplicate of #46
